### DISCUSSION TITLE
mMOTA computation

### DISCUSSION THREAD

**Discussion by alessiasaggese** [bdd100k/bdd100k#231] [Q&A] [answered]
1. How is mMOTA computed?
From the challenge: mean Multiple Object Tracking Accuracy across all 8 categories.
So, is it a weighted mean (dependent on the number of samples per each category) or is it just a mean (independent on the number of samples per each category)?

2. the computation of false positive, misses etc for computing the MOTA score depends on a threshold distance between the GT bounding box and the object bounding box. How is the distance fixed?

**Comment by siyuanliii** [comment, marked as answer]
1. The mMOTA is a mean which is independent on the number of samples per category. You can see the detailed eval code here: https://github.com/scalabel/scalabel/blob/1b13f3b0bd78100cff6d255d7cf28d7f03cb2559/scalabel/eval/mot.py#L259
2. We use the standard MOTA. Thus, the matching is performed by the Hungarian algorithm. You can refer the repo for the details. https://github.com/glisanti/CLEAR-MOT
